docs/installation-configuration.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Astra Setup
+meta_desc: Information on how to install the Astra provider.
+layout: installation
+---
+
+## Installation
+
+The Pulumi Astra provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumiserve/pulumi-astra`](https://www.npmjs.com/package/@pulumiverse/astra)
+* Python: [`pulumi_astra`](https://pypi.org/project/pulumi-astra/)
+* Go: [`github.com/pulumiverse/pulumi-astra/sdk/go/astra`](https://pkg.go.dev/github.com/pulumiverse/pulumi-astra/sdk)
+* .NET: [`Pulumi.Astra`](https://www.nuget.org/packages/Pulumi.Astra)
```

Comment:
The namespace should also be `Pulumiverse`. See here how to configure the .NET namespace:

https://github.com/pulumiverse/infra#nuget